Terminal.Gui/Configuration/ThemeManager.cs

```diff
@@ -55,153 +48,127 @@ namespace Terminal.Gui;
-
-	private static string _theme = string.Empty;
-
-	/// <summary>
-	/// The currently selected theme. This is the internal version; see <see cref="Theme"/>.
-	/// </summary>
-	[JsonInclude, SerializableConfigurationProperty (Scope = typeof (SettingsScope), OmitClassName = true), JsonPropertyName ("Theme")]
-	internal static string SelectedTheme {
-		get => _theme;
-		set {
-			var oldTheme = _theme;
-			_theme = value;
-			if (oldTheme != _theme &&
-				ConfigurationManager.Settings! ["Themes"]?.PropertyValue is Dictionary<string, ThemeScope> themes &&
-				themes.ContainsKey (_theme)) {
-				ConfigurationManager.Settings! ["Theme"].PropertyValue = _theme;
-				Instance.OnThemeChanged (oldTheme);
-			}
-		}
-	}
-
-	/// <summary>
-	/// Gets or sets the currently selected theme. The value is persisted to the "Theme"
-	/// property.
-	/// </summary>
-	[JsonIgnore]
-	public string Theme {
-		get => ThemeManager.SelectedTheme;
-		set {
-			ThemeManager.SelectedTheme = value;
-		}
-	}
-
-	/// <summary>
-	/// Called when the selected theme has changed. Fires the <see cref="ThemeChanged"/> event.
-	/// </summary>
-	internal void OnThemeChanged (string theme)
-	{
-		Debug.WriteLine ($"Themes.OnThemeChanged({theme}) -> {Theme}");
-		ThemeChanged?.Invoke (this, new ThemeManagerEventArgs (theme));
-	}
-
-	/// <summary>
-	/// Event fired he selected theme has changed.
-	/// application.
-	/// </summary>
-	public event EventHandler<ThemeManagerEventArgs>? ThemeChanged;
-
-	/// <summary>
-	/// Holds the <see cref="ThemeScope"/> definitions. 
-	/// </summary>
-	[JsonInclude, JsonConverter (typeof (DictionaryJsonConverter<ThemeScope>))]
-	[SerializableConfigurationProperty (Scope = typeof (SettingsScope), OmitClassName = true)]
-	public static Dictionary<string, ThemeScope>? Themes {
-		get => ConfigurationManager.Settings? ["Themes"]?.PropertyValue as Dictionary<string, ThemeScope>; // themes ?? new Dictionary<string, ThemeScope> ();
-		set {
-			//if (themes == null || value == null) {
-			//	themes = value;
-			//} else {
-			//	themes = (Dictionary<string, ThemeScope>)DeepMemberwiseCopy (value!, themes!)!;
-			//}
-			ConfigurationManager.Settings! ["Themes"].PropertyValue = value;
-		}
-	}
-
-	internal static void Reset ()
-	{
-		Debug.WriteLine ($"Themes.Reset()");
-		Colors.Reset ();
-		Themes?.Clear ();
-		SelectedTheme = string.Empty;
-	}
-
-	internal static void GetHardCodedDefaults ()
-	{
-		Debug.WriteLine ($"Themes.GetHardCodedDefaults()");
-		var theme = new ThemeScope ();
-		theme.RetrieveValues ();
-
-		Themes = new Dictionary<string, ThemeScope> (StringComparer.InvariantCultureIgnoreCase) { { "Default", theme } };
-		SelectedTheme = "Default";
-	}
-
-	#region IDictionary
+    static ThemeManager () { } // Make sure it's truly lazy
+    private ThemeManager () { } // Prevent instantiation outside
+    private static string _theme = string.Empty;
+
+    /// <summary>Holds the <see cref="ThemeScope"/> definitions.</summary>
+    [JsonInclude]
```

Comment:
(out of scope)

Statics included in serialization can have unexpected consequences.